api/types/tunnel.go

```diff
@@ -196,15 +185,37 @@ func (r *ReverseTunnelV2) Check() error {
 	}
 
 	for _, addr := range r.Spec.DialAddrs {
-		_, err := utils.ParseAddr(addr)
-		if err != nil {
+		if _, err := utils.ParseAddr(addr); err != nil {
 			return trace.Wrap(err)
 		}
 	}
 
 	return nil
 }
 
+// TunnelType is the type of tunnel. Either node or proxy.
```

Comment:
```suggestion
// TunnelType is the type of tunnel.
```
I think there're more tunnel types which rendered the comment obsolete.